src/multiselect-combo-box.html

```diff
@@ -7,42 +7,100 @@
 
 <dom-module id="multiselect-combo-box">
   <template>
-    <label part="label">[[label]]</label>
 
-    <div part="readonly-container" hidden$="[[!readonly]]">
-      [[_getReadonlyValue(selectedItems, itemLabelPath, compactMode)]]
-    </div>
+    <style>
+
```

Comment:
Could you please remove this empty line, the linter complains here. Thank you :)